platform/sync-installer/sync-installer-impl/src/main/java/org/codice/ddf/sync/installer/impl/WaitForReadyCommand.java

```diff
@@ -41,6 +41,15 @@
   )
   boolean increaseLogLevel = false;
 
+  @Option(
+    name = "-wt",
+    aliases = {"--wait-time"},
+    description = "Amount of time to wait for the system to reach a ready state in milliseconds.",
```

Comment:
❓ Should it be in seconds or minutes? Since this is a console parameter, it seems weird to ask the user to compute its time in milliseconds.